packages/wdio-browser-runner/src/vite/plugins/esbuild.ts

```diff
@@ -30,5 +30,5 @@ export function codeFrameFix () {
                 }
             )
         }
-    }
+    } as any
```

Comment:
The reason for using "any" here is that during the compilation of the ESM package, this module encounters the error "Types of property 'setup' are incompatible."

![Screenshot 2024-02-18 at 10 04 47 AM](https://github.com/webdriverio/webdriverio/assets/49207189/53f9ffe2-8a6d-48a0-8726-4e9c668323d5)

Reply:
This is because the Esbuild version we use as dependency is different that the one used by Vite. This is fixed in `main` so I don't mind keeping as is.